cleanTranslations.js

```diff
@@ -0,0 +1,53 @@
+const fs = require('fs-extra');
+const glob = require('glob');
+
+const PROJECT_DIR = './';
+const TRANSLATION_DIR = './apps/web/public/static/locales';
+const regex = /([`'"])(.*?)\1/g;
+const regexConcatenated = /t\(\s*(['"`])([^'"`]*?)\${[^}]*}[^'"`]*?\1\s*\)/g;
+const regexTemplateLiteral = /\[`?\$\{t\((['"`])(.*?)\1\)\}\]?/g;
```

Comment:
Please add comments for each regex. It will make it easier for us to understand the regex and debug in future.